app/models/claims/claim.rb

```diff
@@ -28,8 +28,8 @@
 #  fk_rails_...  (provider_id => providers.id)
 #  fk_rails_...  (school_id => schools.id)
 #
-class Claim < ApplicationRecord
-  belongs_to :school, class_name: "Claims::School"
+class Claims::Claim < ApplicationRecord
+  belongs_to :school
   belongs_to :provider
```

Comment:
If we have Claims::Provider, this be 
```suggestion
  belongs_to :provider, class_name: "Claims::Provider"
```

Reply:
Same as above, the namespace is implicit.

Reply:
Too much rails magic for me